Arduino/SensorLogic/SensorLogic.ino

```diff
@@ -30,7 +28,12 @@ void Fingerprintsetup(void);
 void Fingerprintloop(void);
 void RFIDsetup(void);
 void RFIDloop(void);
+void Servoloop(void);
+void Servosetup(void);
+
+Servo myservo;
 
+int pos = 90;
```

Comment:
pos is the "position" of the servo. However, the AR-360HB is a continuous servo motor. Thus, it serves more as a speed setting.

- 0 is max speed in one direction
- 90 is no movement
- 180 is max speed in other direction